spec/controllers/vat_rates_controller_spec.rb

```diff
@@ -13,13 +13,10 @@
 
 RSpec.describe VatRatesController, type: :controller do
 
-  before(:all) do
-    @vr1 = FactoryBot.create :vat_rate, effective_date: Date.new(2000, 1, 1),  rate_base_points: 1750
-    @vr2 = FactoryBot.create :vat_rate, effective_date: Date.new(2011, 4, 1),  rate_base_points: 2000
-  end
-
-  after(:all) do
-    VatRate.destroy( [ @vr1.id, @vr2.id ] )
+  before do
+    VatRate.delete_all
```

Comment:
❓  Don't have much knowledge or background on this (so bare with me 😅 ), but since this is now a `before(:each)` block what's the reason for deleting all vat rates each time? (they should be transitional)

Reply:
No worries, there is one seeded VatRate for the whole suite (see before(:suite) in rails helper) so to make the test more explicit/clear in this spec, in particular,  I have removed it so you can be sure you only have the VAT rates the test defines.

I have removed the before(:all) because it is [dodgy](https://makandracards.com/makandra/11507-using-before-all-in-rspec-will-cause-you-lots-of-trouble-unless-you-know-what-you-are-doing) and causes flickers from my experience. could put the `delete_all` in a before(:all), but i think any performance/speed boost we may get from it carries the aforementioned risk